doc/en/user/source/community/elasticsearch/index.rst

```diff
@@ -170,6 +168,8 @@ The initial layer configuration panel for an Elasticsearch layer will include an
 
 To return to the field table after it has been closed, click the "Configure Elasticsearch fields" button below the "Feature Type Details" panel on the layer configuration page.
 
+The ``Short Names`` feature has been removed as it is not compatible with Elasticsearch 2.0 and beyond. Previous fields which used the short names will be reverted to the full name, but you can still use aliasing to accomplish the same effect.
```

Comment:
Consider moving this note down to the [faq](https://github.com/geoserver/geoserver/blob/8678a31902381bf13ec7bb3808a4d1f4985301e8/doc/en/user/source/community/elasticsearch/index.rst#id1) section where there are other caveats and upgrade notes. Maybe preface with something like "when updating from ElasticGeo 2.16.0 note that ...".

Reply:
Sounds good. Done.